servers/Azure.Mcp.Server/server.json

```diff
@@ -0,0 +1,51 @@
+{
+    "$schema": "https://static.modelcontextprotocol.io/schemas/2025-10-17/server.schema.json",
+    "name": "com.microsoft/azure-mcp",
+    "description": "All Azure MCP tools to create a seamless connection between AI agents and Azure services.",
+    "version": "2.0.0-beta.2",
+    "title": "Azure MCP Server",
```

Comment:
The title appears to be `Azure` right now, not `Azure MCP Server` (assuming I am guessing the upcoming `title` plumbing property)
<img width="2052" height="711" alt="image" src="https://github.com/user-attachments/assets/48a26baa-1969-4e3b-823d-c0541a643d80" />